CarbonFight/actions
# chore: add jest setup to run test in local env

## How to test: 
- Run `just dev` in a console.
- In another console, run `just test-init` then `just test`.
- You should see a passing test ! :tada:
